src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KafkaClientFetchFactory.java

```diff
@@ -1216,6 +1230,37 @@ private int decodeFetchRecord(
         return progress;
     }
 
+    private boolean matchFilters(
+        KafkaFetchStream.KafkaFetchClient client,
+        long offsetAbs,
+        Array32FW<KafkaFilterFW> filters)
+    {
+        AtomicBoolean matchesFilters = new AtomicBoolean(false);
+        filters.forEach(f ->
+        {
+            AtomicBoolean matchesConditions = new AtomicBoolean(true);
+            f.conditions().forEach(c ->
+            {
+                switch (c.kind())
+                {
+                case KIND_AGE:
+                    KafkaAge age = c.age().get();
+                    if (age == LIVE)
+                    {
+                        if (offsetAbs < client.latestOffset)
+                        {
+                            matchesConditions.set(false);
+                            break;
+                        }
+                    }
+                    break;
+                }
+            });
+            matchesFilters.set(matchesFilters.get() | matchesConditions.get());
+        });
+        return matchesFilters.get();
+    }
+
```

Comment:
We don't support filters on the client fetch factory (only in cache), so no need to filter by age here.